test/compare_benchmarks.py

```diff
@@ -0,0 +1,91 @@
+import os
+import glob
+import argparse
+import multiprocessing as mp
+
+import thicket as tt
+
+# Note: script does not work for advection_reaction_3D benchmarks currently.
```

Comment:
Is this still true? If we know why, it would be good to note that here as well.

Reply:
It should work now with the closure of the hatchet issue. ive removed the comment